spec/models/schedule_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Schedule, type: :model do
+  describe 'validation' do
```

Comment:
Should probably add an `valid with valid attributes` entry too. Just for a sanity check.